addons/product/tests/test_product_rounding.py

```diff
@@ -0,0 +1,85 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.tests import tagged
+from odoo.addons.product.tests.common import ProductCommon
+from odoo.fields import Command
+import time
```

Comment:
```suggestion
import time

from odoo.fields import Command
from odoo.tests import tagged

from odoo.addons.product.tests.common import ProductCommon
```

nitpicking (should have told you with my previous review, sorry), but we try to keep the imports ordered according to our [guidelines](https://www.odoo.com/documentation/master/contributing/development/coding_guidelines.html#imports) ;)

Reply:
Just amended this to the commit, thanks again for the pointers!